I0701 19:23:39.809396 130302610413376 03_eval_finetuned_copy.py:52] Loading finetuned model...
Loading checkpoint shards: 100%|█████████████████████████████████████████████████| 3/3 [00:02<00:00,  1.42it/s]
Traceback (most recent call last):
  File "/home/yunqiliu/octo/examples/03_eval_finetuned_copy.py", line 137, in <module>
    app.run(main)
  File "/home/yunqiliu/anaconda3/envs/torch/lib/python3.11/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/home/yunqiliu/anaconda3/envs/torch/lib/python3.11/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
             ^^^^^^^^^^
  File "/home/yunqiliu/octo/examples/03_eval_finetuned_copy.py", line 59, in main
    model_vla = AutoModelForCausalLM.from_pretrained(vla_args.model_name_or_path, **model_kwargs).to(device)
                                                                                                     ^^^^^^
NameError: name 'device' is not defined